sdk/scheduler/src/test/java/com/mesosphere/sdk/scheduler/DefaultSchedulerTest.java

```diff
@@ -872,6 +872,16 @@ private void statusUpdate(Protos.TaskID launchedTaskId, Protos.TaskState state)
         taskIds.add(installStep(1, 0, getSufficientOfferForTaskB()));
         taskIds.add(installStep(1, 1, getSufficientOfferForTaskB()));
 
+        while (true) {
```

Comment:
`while (!defaultScheduler.deploymentPlanManager.getPlan().isComplete()) { ...sleep... }`?

Could also use a for loop to explicitly limit the number of iterations within the test:

`for (int i = 0; i < 10 && !defaultScheduler.deploymentPlanManager.getPlan().isComplete(); ++i) { ... sleep ...}`